DailyWeather/Controllers/CBDWeatherFetcher.m

```diff
@@ -0,0 +1,62 @@
+//
+//  CBDWeatherFetcher.m
+//  DailyWeather
+//
+//  Created by Christopher Devito on 5/14/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+#import "CBDWeatherFetcher.h"
+#import "LSIErrors.h"
+#import "CBDWeatherForecast.h"
+
+@implementation CBDWeatherFetcher
+
+static NSString *baseURLString = @"https://api.darksky.net/forecast/18990986362b5b52af4a81dd7775c5af";
+// https://api.darksky.net/forecast/[key]/[latitude],[longitude]
+
+- (void)fetchWeatherAtLatitude:(double)latitude
+                     longitude:(double)longitude
+               completionBlock:(CBDWeatherFetcherCompletion)completionBlock {
+
+    NSLog(@"Fetching...");
+
+    
+    NSString *locationCoordinates = [NSString stringWithFormat:@"%f,%f", latitude, longitude];
```

Comment:
Hmm, since this is UI-related (a string representation of two doubles, essentially), it might be best to put it on the view controller so as not to confuse things. Might also be good to call it something even more descriptive, like `locationCoordinatesDisplayText`.